speed-dreams/source-2.2.3/src/modules/userinterface/legacymenu/mainscreens/DeveloperMenu.cpp

```diff
@@ -23,6 +23,15 @@
 
 #define MSG_CHOOSE_REPLAY_NORMAL_TEXT "Choose Replay File: "
 
+#define SET_DEFAULT_THRESHOLDS(accel, brake, steer) \
+    m_decisionThresholds.Accel = accel;             \
+    m_decisionThresholds.Brake = brake;             \
+    m_decisionThresholds.Steer = steer
+
+#define SET_DEFAULT_THRESHOLD_ONLY_SIGNALS      SET_DEFAULT_THRESHOLDS(1, 0.9f, 0.09f);
```

Comment:
You might want to opt to not include the semicolon here, but that's personal choice entirely. Otherwise, PR accepted.